certified-connectors/OutlookTasks/apiDefinition.swagger.json

```diff
@@ -0,0 +1,481 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Outlook Tasks",
+    "description": "Outlook Tasks service lets you create, read, synchronize, update and delete your tasks that are secured by Azure Active Directory in Office 365 or a Microsoft account."
+  },
+  "host": "outlook.office.com",
+  "basePath": "/api/v2.0",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/actions/GetAllTaskFolders/v2/me/taskfolders": {
+      "get": {
+        "tags": [
+          "Tasks"
+        ],
+        "summary": "List all task folders",
+        "description": "Returns a list of all the task folders in your mailbox.",
+        "operationId": "GetAllTaskFolders_V2",
+        "parameters": [
+          {
+            "$ref": "#/parameters/OdataQueryOption-$top"
+          },
+          {
+            "$ref": "#/parameters/OdataQueryOption-$skip"
+          }
+        ],
+        "responses": {
+          "200": {
+            "$ref": "#/responses/OdataCollection<TaskFolder>"
+          }
+        },
+        "x-ms-api-annotation": {
+          "family": "GetAllTaskFolders",
+          "revision": 2
+        }
+      }
+    },
+    "/actions/CreateTaskFolder/v2/me/taskfolders": {
+      "post": {
+        "tags": [
+          "Tasks"
+        ],
+        "summary": "Create a task folder",
+        "description": "This operation creates a new task folder in the default task group (`My Tasks`) in your mailbox.",
```

Comment:
Should `My Tasks` be the single quotes 'My Tasks'?

Reply:
`` are MD syntax, which is what's supported in these description properties. Note how even your comment got the special syntax highlighting.